index.html

```diff
@@ -0,0 +1,240 @@
+                <ul class="lan-list">
+                    <li class="lan-item">Bootstrap</li>
+                    <li class="lan-item">Ruby on Rails</li>
+                    <li class="lan-item">RSPec</li>
+                    <li class="lan-item">CapyBara</li>
+                    <li class="lan-item">Selenium</li>
+                </ul>
+            </li>
+            <li class="sub-lan">
+                <h2 class="lan-title"><img class="lan-span" alt="cat icon" src="images/3.png"></span>Skills</h2>
+                <ul class="lan-list">
+                    <li class="lan-item">Codekit</li>
+                    <li class="lan-item">Github</li>
+                    <li class="lan-item">Codepen</li>
+                    <li class="lan-item">Gitlab</li>
+                    <li class="lan-item">Terminal</li>
+                </ul>
+            </li>
+        </ul>
+    </section>
+    <section class="contact">
+        <form class="contact-form" action="https://formspree.io/f/xnqlkdkv" method="POST">
+            <p class="contact-para">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <ul class="ins">
+                <li class="in">
+                    <input class="contact-input" type="text" id="name" name="name" placeholder="username" maxlength="30" required>
+                </li>
+                <li class="in">
+                    <input class="contact-input" type="email" id="email" name="email" placeholder="user@email.com" required>
+                </li>
+                <li class="in">
+                    <textarea class="message-input" id="msg" name="message" maxlength="500" placeholder="Enter your Message here" required></textarea>
+                </li>
+                <div class="btncontainer"> <button class="contact-btn" type="submit" name="button">Get in touch</button>
+                </div>
+            </ul>
+            <hr>
+        </form>
+        <ul class="contact-icons">
+            <li>
+                <a href="" >
+                    <img class="contact-icon" alt="cat icon" src="images/Vector1.png">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img class="contact-icon" alt="linkedin icon" src="images/Vector2.png">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img class="contact-icon" alt="2 icon" src="images/Vector3.png">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img class="contact-icon" alt="t icon" src="images/Vector4.png">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img class="contact-icon" alt="m icon" src="images/Vector5.png">
+                </a>
+            </li>
+        </ul>
```

Comment:
Please do not repeat code twice you can use one code for both sections ,because like this you just increase th code line for nothing.